README.md

```diff
@@ -6,6 +6,27 @@
 
 John has worked on a number of projects for me over the past **three years**. His contributions have been **significant** to the successful deployment of an **enterprise** document publishing system. His skills in **XSLT**, **XSD** and **XML**, **Ruby** and **Python**, combine with a **design approach** that results in **compact** and **agile** code. His **"outside-the-box"** solutions have also proven their worth with ongoing maintenance and support.
 
+### Software engineer, GitHub Developer maintainer
+
+September 2018 – Present.  Iran.
+
+**Max** is a software engineer or maybe just a programmer, and a **GitHub Developer** maintainer, so far creator of **The One programming language**.
+His life is mostly as a computer lover with a focus on compiler and programming languages, It's why he tested most of the tools and languages. Anyway, He has very good experience in Web development and software design. His educational background is **mathematics** field at the University of Kashan. It's a mathematical user by the way he connected to a community of CE and CS students and community.
+
+You maybe think why do a mathematical student and person work in computer science, Yes. it may be a bit strange. 
+He also researches with professors of the university to do some research and attends Science conferences (If he finds some time, since he is mostly busy with Open-Source projects of the team) and its mission is to combine mathematics science with the computer. Anyway, He is a programmer and not a mathematics man.
```

Comment:
```suggestion
`Max` also researches with professors of the university, attends science conferences (If he finds some time, since he is mostly busy with Open-Source projects) and his mission is to combine mathematics science with the computer. Anyway, `Max` is a programmer and not a mathematician.
```